source/import-pipeline-from-text.txt

```diff
@@ -24,18 +27,39 @@ pipeline stages using the Pipeline Builder's :guilabel:`Output` panes.
 Syntax
 ------
 
-The imported pipeline must be in the
-:manual:`MongoDB query language </tutorial/query-documents/>`
-(i.e. the same syntax used as the ``pipeline`` parameter of the
-:manual:`db.collection.aggregate()
-</reference/method/db.collection.aggregate/>` method). The imported
-pipeline must be an array, even if there is only one stage in the
-pipeline.
+The imported pipeline must be:
+
+- In the :manual:`MongoDB query language </tutorial/query-documents/>`
+  (i.e. the same syntax used as the ``pipeline`` parameter of the
+  :manual:`db.collection.aggregate()
+  </reference/method/db.collection.aggregate/>` method).
+- Must be an array, even if there is only one stage in the pipeline.
 
 Procedure
 ---------
 
-.. include:: /includes/steps/import-pipeline-from-text.rst
+.. procedure::
+   :style: normal
+
+   .. step:: In the aggregation pipeline pane, click the 
+      :guilabel:`</> TEXT` toggle switch to enable text mode for 
+      pipeline editing.
+
+      .. figure:: /images/compass/agg-builder-textview-toggle-on.png
+         :alt: Toggle Textview Mode on
+
+   .. step:: Type or paste aggregation pipeline syntax text into the 
+      text editor. The text editor provides real time linting 
+      for correct syntax and debugging information.
+
+      .. code-block:: javascript
+
+         [ { "$limit" : 4 } ]
```

Comment:
This code-block feels slightly out of context as written. Can we provide a lead-in with something like:

```
For example, the following pipeline consists of a single :pipeline:`$limit` stage: 
```